scripts/check_configs.ts

```diff
@@ -420,6 +426,39 @@ function isValidMarketPromo(marketPromo: {[marketId: string]: MarketPromo}, netw
   return true;
 }
 
+function isValidQuickSelectTokens(quickSelectTokens: TransferQuickSelectToken[], network: CarbonSDK.Network, denoms: string[]): boolean {
+  const duplicateQuickSelectTokens = checkDuplicateEntries(quickSelectTokens.map(token => token.show));
+  const invalidQuickSelectTokens = checkValidEntries(quickSelectTokens.map(token => token.show), denoms);
+
+  const duplicateRealTokens = checkDuplicateEntries(quickSelectTokens.map(token => token.real));
+  const invalidRealTokens = checkValidEntries(quickSelectTokens.map(token => token.real), denoms);
+
+  if (duplicateQuickSelectTokens.status && duplicateQuickSelectTokens.entry) {
+    let listOfDuplicates: string = duplicateQuickSelectTokens.entry.join(", ");
+    console.error(`ERROR: ${network}.json has the following duplicated quick select tokens: ${listOfDuplicates}. Please make sure to input each token only once in ${network}`);
+    return false;
+  }
+
+  if (invalidQuickSelectTokens.status && invalidQuickSelectTokens.entry) {
+    let listOfInvalidTokens: string = invalidQuickSelectTokens.entry.join(", ");
+    console.error(`ERROR: ${network}.json has the following invalid quick select token denoms: ${listOfInvalidTokens}. Please make sure to only input valid token denoms in ${network}`);
+    return false;
+  }
+
+  if (duplicateRealTokens.status && duplicateRealTokens.entry) {
+    let listOfDuplicates: string = duplicateRealTokens.entry.join(", ");
+    console.error(`ERROR: ${network}.json has the following duplicated real tokens: ${listOfDuplicates}. Please make sure to input each token only once in ${network}`);
+    return false;
+  }
+
+  if (invalidRealTokens.status && invalidRealTokens.entry) {
+    let listOfInvalidTokens: string = invalidRealTokens.entry.join(", ");
+    console.error(`ERROR: ${network}.json has the following invalid real token denoms: ${listOfInvalidTokens}. Please make sure to only input valid token denoms in ${network}`);
+    return false;
```

Comment:
i think just need to check if `quick_select_denom` is a valid token should be sufficient. Different tokens can still point to the same selected token 
i.e.  USDC, axlUSDC can all point to cgt/1 

but do check for duplicate labels e.g. cannot have two USDC buttons appearing